### DISCUSSION TITLE
Problems with OrientationMarkerWidget and AxesActor

### DISCUSSION THREAD

**Discussion by dougbrennan** [Kitware/trame#509] [General] [answered]
Hello!!!,

I'm trying to add axes to my scene and I'm running into problems.

I'm setting up lin this

```
        self.triad_orientation_widget = vtkOrientationMarkerWidget()
        self.triad_actor = vtkAxesActor()
        self.triad_orientation_widget.SetOutlineColor(0.9300, 0.5700, 0.1300)
        self.triad_orientation_widget.SetOrientationMarker(self.triad_actor)
        self.triad_orientation_widget.SetInteractor(self.interactor)
        self.triad_orientation_widget.SetViewport(0.0, 0.0, 0.15, 0.15)
        self.triad_orientation_widget.SetEnabled(1)
        self.triad_orientation_widget.InteractiveOn()   
``` 

And I'm displaying in a VtkRemoteLocalView.

- WIth mode=remote, it's almost working. The widget and the axes display and rotate but there's a bunch of extra large text draw on display as shown below. 
- With mode=local, there's no extra text, but the triad doesn't rotate....

Any ideas? 
![image](https://github.com/Kitware/trame/assets/66567799/b81eec23-b320-4f06-b825-22d68eda245f)

**Comment by dougbrennan** [comment, marked as answer]
Ok - Just found a response to a previous from Sebastien noting that this is a limitation on VTK.js......  
My bad for not seeing the comment before I posted. I searched for "triad" and it came up blank - should have searched for axes....

**Comment by jourdain** [comment]
The orientation axis should work with the local view (no text label though). But you need to register it as widget. You can see how we do it in PyVista.
